feature/one-more-file.txt

```diff
@@ -0,0 +1,4 @@
+This file was also added in feature-1 branch
```

Comment:
It's really good that you added this line of code

Reply:
Thanks for this comment